src/index.ts

```diff
@@ -8,6 +8,7 @@ import alignTransform from './transforms/align';
 import indentTransform from './transforms/indent';
 import crlfTransform from './transforms/crlf';
 import { flow as flowTransform } from './transforms/index';
+import { rewireSpecs, rewireSource, seedBlock, seedTokens } from './util';
```

Comment:
Here's a more succinct alternative, assuming TS supports it:

```suggestion
export { rewireSpecs, rewireSource, seedBlock, seedTokens } from './util';
```

Then you can get rid of the other export.

Reply:
if I read it right, all entities will be exported as top-level names which I was explicitly trying to avoid

Reply:
Oh yes, sorry, I see why you were doing that. Right, you can just leave it all as is then.

Reply:
cool, merging then